src/script/page/MainContent/panels/preferences/OptionPreferences.tsx

```diff
@@ -161,7 +183,15 @@ const OptionPreferences: React.FC<OptionPreferencesProps> = ({
 
           <hr className="preferences-separator" />
 
-          <PreferencesSection title={t('preferencesOptionsPopular')}>
+          <PreferencesSection title={t('preferencesOptionsAppearance')}>
+            <div css={{marginBottom: '1.5rem', width: '100%'}}>
```

Comment:
why are we using inline CSS?

Reply:
I should probably raise that on the web chat, I'm under the impression we're moving toward inline CSS for oneoff rules like this

Reply:
I'm personally, not a big fan of inline css. 
I'm not formally against it either, but when possible it's be nice to extract those into a `Component.styles.ts`